src/main/java/org/prebid/server/bidder/sharethrough/SharethroughBidder.java

```diff
@@ -26,19 +28,24 @@
 import org.prebid.server.util.HttpUtil;
 
 import java.io.IOException;
+import java.text.SimpleDateFormat;
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.Date;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 import java.util.stream.Collectors;
 
-public class SharethroughBidder implements Bidder<Void> {
+public class SharethroughBidder implements Bidder<SharethroughRequestBody> {
 
     private static final String VERSION = "4";
     private static final String SUPPLY_ID = "FGMrCMMc";
     private static final String DEFAULT_BID_CURRENCY = "USD";
     private static final BidType DEFAULT_BID_TYPE = BidType.xNative;
+    private static final Long DEFAULT_TIMEOUT = 10L;
+    private static final Date TEST_TIME = new Date(1604455678999L);
```

Comment:
Could you help me figure out where did this come from? I can only see the date they set in `butler_test`, is it the same one?

Reply:
I need this `date` only for test bc they use current time and use it in request body and url parameter. 
It is the only solution that I came out with. (or we can delete integrational test)